index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+         
+</script>
+    </head>
+    <body>
+        
+        <div id = title>
+           <h2>Project One</h2> 
+        </div>
+
+        
+        <div class="grading-element-five">
```

Comment:
very close...`document.querySelector("#grading-element-five")` returns `null`...why? You're very very close.